src/api/common/utils/EntityUtils.ts

```diff
@@ -60,9 +60,41 @@ export const POST_MULTIPLE_LIMIT = 100
  */
 export type Stripped<T extends Partial<SomeEntity>> = Omit<
 	T,
-	"_id" | "_ownerGroup" | "_ownerEncSessionKey" | "_permissions" | "_errors" | "_format" | "_type" | `_finalEncrypted${string}` | `_defaultEncrypted${string}`
+	| "_id"
+	| "_area"
+	| "_owner"
+	| "_ownerGroup"
+	| "_ownerEncSessionKey"
+	| "_permissions"
+	| "_errors"
+	| "_format"
+	| "_type"
+	| `_finalEncrypted${string}`
+	| `_defaultEncrypted${string}`
 >
 
+type OptionalEntity<T extends Entity> = T & {
```

Comment:
I think both names are a bit confusing?

As I understand, `OptionalEntity` is for cases where we didn't create the entity on the server yet and `StrippedEntity` is entity without technical fields.

Reply:
Suggestions are welcome. `StrippedEntity` is just `Stripped` but for any `Entity`, rather than just for `SomeEntity`s.

`OptionalEntity` is a `StrippedEntity` you may add technical fields to (because typescript wouldn't let us `_ownerKey`  to a `StrippedEntity`, for example).